libsql/internal/ws/driver.go

```diff
@@ -86,21 +86,29 @@ func convertToNamed(args []driver.Value) []driver.NamedValue {
 }
 
 func (s stmt) Exec(args []driver.Value) (driver.Result, error) {
-	ctx, cancel := context.WithTimeout(context.Background(), defaultWSTimeout)
-	defer cancel()
-	return s.c.ExecContext(ctx, s.query, convertToNamed(args))
+	return s.c.ExecContext(context.Background(), s.query, convertToNamed(args))
 }
 
 func (s stmt) Query(args []driver.Value) (driver.Rows, error) {
-	ctx, cancel := context.WithTimeout(context.Background(), defaultWSTimeout)
-	defer cancel()
-	return s.c.QueryContext(ctx, s.query, convertToNamed(args))
+	return s.c.QueryContext(context.Background(), s.query, convertToNamed(args))
```

Comment:
Maybe just call s.QueryContext(ctx, args)?

Reply:
hey, I wanted to follow the convention of non-ctx methods internally calling ctx ones, they just act like a simple wrappers, without of any logic. Say in future, if you make changes to `s.c.ExecContext`, then you'd have to update it at two places instead of one.

However, if you still think it needs to be changed then I am happy to do it.

Reply:
oh right, I got confused. 

What you said is exactly what I am doing, but in the last commit. Let me squash the commits, then it will be easier to follow